packages/browser-lite/src/mesh.ts

```diff
@@ -129,6 +129,81 @@ window.addEventListener(loadEventName, () => {
     isWasmLoaded = true;
 });
 
+const addressSchema = {
+    $id: 'http://example.com/address',
+    type: 'string',
+    pattern: '^0x[0-9a-fA-F]{40}$',
+};
+const wholeNumberSchema = {
+    $id: 'http://example.com/wholeNumber',
+    anyOf: [{ type: 'string', pattern: '^\\d+$' }, { type: 'integer' }],
+};
+const hexSchema = { $id: 'http://example.com/hex', type: 'string', pattern: '^0x(([0-9a-fA-F][0-9a-fA-F])+)?$' };
```

Comment:
Do we have to change the `$id` fields here compared to the standalone Go implementation? We specifically chose an order for compiling the JSON schema such that common types like `/address` and `/wholeNumber` are exposed and can be used in custom order filters. Changing this technically breaks backwards compatibility and breaks compatibility between the standalone and browser nodes.

Reply:
It turned out that we didn't! I needed to use `addSchema` instead of passing them into the constructor, but it appears to work like a charm now.